lib/core/src/lib/form/components/form-renderer.component.html

```diff
@@ -16,56 +19,43 @@
                         <mat-icon>{{ currentRootElement?.isExpanded ? 'expand_more' : 'expand_less' }}</mat-icon>
                     </button>
-                    <span
-                        (click)="onExpanderClicked(currentRootElement)"
-                        role="button"
-                        tabindex="0"
-                        (keyup.enter)="onExpanderClicked(currentRootElement)"
-                        [id]="'container-header-label-' + currentRootElement?.id"
-                        >{{ currentRootElement.name | translate }}</span
-                    >
+                    <span (click)="onExpanderClicked(currentRootElement)"
+                          [id]="'container-header-label-'+currentRootElement?.id">
+                        {{ currentRootElement.name | translate }}
+                    </span>
                 </h4>
             </div>
 
-            <div *ngIf="currentRootElement?.form?.enableFixedSpace; else fixingTemplate">
-                <div
-                    class="adf-grid-list"
-                    [ngStyle]="{ 'grid-template-columns': 'repeat(' + getNumberOfColumns(currentRootElement) + ', 1fr)' }"
-                    *ngIf="currentRootElement?.isExpanded"
-                >
-                    <div
-                        class="adf-grid-list-item"
-                        *ngFor="let field of getContainerFields(currentRootElement)"
-                        [ngStyle]="{ 'grid-area': 'auto / auto / span ' + (field?.rowspan || 1) + ' / span ' + (field?.colspan || 1) }"
-                    >
+            <div *ngIf="currentRootElement?.form?.enableFixedSpace else fixingTemplate">
+                <div class="adf-grid-list"
+                     [ngStyle]="{ 'grid-template-columns': 'repeat(' + getNumberOfColumns(currentRootElement) + ', 1fr)'}"
+                     *ngIf="currentRootElement?.isExpanded">
+                    <div class="adf-grid-list-item" *ngFor="let field of getContainerFields(currentRootElement)"
+                         [ngStyle]="{ 'grid-area': 'auto / auto / span ' + (field?.rowspan || 1) + ' / span ' + (field?.colspan || 1) }">
                         <adf-form-field *ngIf="field" [field]="field"></adf-form-field>
                     </div>
                 </div>
             </div>
 
             <ng-template #fixingTemplate>
                 <section class="adf-grid-list-column-view" *ngIf="currentRootElement?.isExpanded">
-                    <div
-                        class="adf-grid-list-single-column"
-                        *ngFor="let column of currentRootElement?.columns"
-                        [style.width.%]="getColumnWith(currentRootElement)"
-                    >
+                    <div class="adf-grid-list-single-column"
+                         *ngFor="let column of currentRootElement?.columns"
+                         [style.width.%]="getColumnWith(currentRootElement)">
```

Comment:
Pre-existing typo, but this method should probably be called 'getColumnWidth()'

Reply:
done